src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -42,6 +42,7 @@ template<typename FitnessFunction = GiniGain,
          template<typename> class NumericSplitType = BestBinaryNumericSplit,
          template<typename> class CategoricalSplitType = AllCategoricalSplit,
          typename DimensionSelectionType = AllDimensionSelect,
+         typename MDLPenaltyType = NoPenalty<FitnessFunction>,
```

Comment:
what does adding MDLPenaltyType have to do with changing default adaboost values?